src/main/java/org/mdbenefits/app/submission/actions/PreCheckPrograms.java

```diff
@@ -0,0 +1,35 @@
+package org.mdbenefits.app.submission.actions;
+
+import formflow.library.config.submission.Action;
+import formflow.library.data.Submission;
+
+import java.util.ArrayList;
+
+import lombok.extern.slf4j.Slf4j;
+import org.mdbenefits.app.data.enums.HelpNeededType;
+import org.mdbenefits.app.utils.ApplicationUtilities;
+import org.springframework.stereotype.Component;
+
+/**
+ *
+ */
+@Component
+@Slf4j
+public class PreCheckPrograms implements Action {
+
+        @Override
+        public void run(Submission submission) {
+                ArrayList<String> helpNeeded = (ArrayList<String>) submission.getInputData().get("helpNeeded[]");
```

Comment:
Here is where I'd just use List again: 

`List<String> helpNeeded = (List) submission.getInputData().get("helpNeeded[]");`